src/3rdparty/get_libs.sh

```diff
@@ -22,8 +22,9 @@ mv libpng-${LIBPNG_VERSION} libpng/src
 # get and apply apng patch
 curl -Lo libpng-apng.patch.gz "https://download.sourceforge.net/project/apng/libpng/libpng16/libpng-${LIBPNG_VERSION}-apng.patch.gz"
 gunzip libpng-apng.patch.gz
+tr -d '\r' < libpng-apng.patch > tmpfile && mv tmpfile libpng-apng.patch
```

Comment:
The APNG patch for 1.6.45 was published with Windows-style line endings which made it not want to apply properly, so this was added to strip any carriage returns in the patch if present.